crates/next-core/src/next_manifests/mod.rs

```diff
@@ -62,7 +62,7 @@ pub struct MiddlewareMatcher {
     // When skipped next.js with fill that during merging.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub regexp: Option<RcStr>,
-    #[serde(skip_serializing_if = "bool_is_true")]
+    #[serde(skip_serializing_if = "bool_is_true", default = "bool_true")]
```

Comment:
It's probably a little less efficient, but if the struct already implements `Default` (like this does), I personally prefer the `default` container attribute, as that guarantees both implementations stay in sync: https://serde.rs/container-attrs.html#default